<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>637</width>
                <height>618</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
                color: rgb(255, 255, 255);
            </string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QGridLayout" name="gridLayout">
                <item row="3" column="4" colspan="2">
                    <widget class="QLabel" name="index_label">
                        <property name="text">
                            <string>Index to Remove:</string>
                        </property>
                    </widget>
                </item>
                <item row="1" column="3">
                    <widget class="QLabel" name="p_label">
                        <property name="text">
                            <string>Password:</string>
                        </property>
                    </widget>
                </item>
                <item row="0" column="0">
                    <spacer name="topSpacer">
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>20</width>
                                <height>40</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item row="7" column="1" colspan="3">
                    <widget class="QListWidget" name="listWidget">
                        <property name="styleSheet">
                            <string notr="true"/>
                        </property>
                        <property name="movement">
                            <enum>QListView::Static</enum>
                        </property>
                        <property name="resizeMode">
                            <enum>QListView::Adjust</enum>
                        </property>
                        <item>
                            <property name="text">
                                <string>None</string>
                            </property>
                        </item>
                    </widget>
                </item>
                <item row="2" column="2">
                    <widget class="QLineEdit" name="usernames">
                        <property name="styleSheet">
                            <string notr="true">color: rgb(255, 255, 255);
                                gridline-color: rgb(0, 0, 0);
                                background-color: rgb(80, 80, 80);
                                border-radius: 5px;
                            </string>
                        </property>
                    </widget>
                </item>
                <item row="1" column="7" rowspan="7">
                    <spacer name="rightSpacer">
                        <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>40</width>
                                <height>20</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item row="4" column="4" colspan="2">
                    <widget class="QLineEdit" name="index_input">
                        <property name="styleSheet">
                            <string notr="true">color: rgb(255, 255, 255);
                                gridline-color: rgb(0, 0, 0);
                                background-color: rgb(80, 80, 80);
                                border-radius: 5px;
                            </string>
                        </property>
                    </widget>
                </item>
                <item row="1" column="1">
                    <widget class="QLabel" name="web_label">
                        <property name="text">
                            <string>Website</string>
                        </property>
                    </widget>
                </item>
                <item row="1" column="0" rowspan="7">
                    <spacer name="leftSpacer">
                        <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>40</width>
                                <height>20</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item row="6" column="6">
                    <widget class="QPushButton" name="update_login">
                        <property name="styleSheet">
                            <string notr="true">color: rgb(255, 255, 255);
                                background-color: rgb(79, 79, 79);
                            </string>
                        </property>
                        <property name="text">
                            <string>Update Login</string>
                        </property>
                    </widget>
                </item>
                <item row="5" column="6">
                    <widget class="QPushButton" name="remove_all_button">
                        <property name="styleSheet">
                            <string notr="true">color: rgb(255, 255, 255);
                                background-color: rgb(79, 79, 79);
                            </string>
                        </property>
                        <property name="text">
                            <string>Remove All</string>
                        </property>
                    </widget>
                </item>
                <item row="2" column="4" colspan="2">
                    <widget class="QPushButton" name="add_button">
                        <property name="styleSheet">
                            <string notr="true">color: rgb(255, 255, 255);
                                background-color: rgb(79, 79, 79);
                            </string>
                        </property>
                        <property name="text">
                            <string>Add User</string>
                        </property>
                    </widget>
                </item>
                <item row="6" column="1">
                    <widget class="QLabel" name="acc_label">
                        <property name="text">
                            <string>Accounts:</string>
                        </property>
                    </widget>
                </item>
                <item row="4" column="6">
                    <widget class="QPushButton" name="remove_user">
                        <property name="styleSheet">
                            <string notr="true">color: rgb(255, 255, 255);
                                background-color: rgb(79, 79, 79);
                            </string>
                        </property>
                        <property name="text">
                            <string>Remove User</string>
                        </property>
                    </widget>
                </item>
                <item row="2" column="1">
                    <widget class="QLineEdit" name="website_input">
                        <property name="styleSheet">
                            <string notr="true">color: rgb(255, 255, 255);
                                gridline-color: rgb(0, 0, 0);
                                background-color: rgb(80, 80, 80);
                                border-radius: 5px;
                            </string>
                        </property>
                    </widget>
                </item>
                <item row="2" column="3">
                    <widget class="QLineEdit" name="passwords">
                        <property name="styleSheet">
                            <string notr="true">color: rgb(255, 255, 255);
                                gridline-color: rgb(0, 0, 0);
                                background-color: rgb(80, 80, 80);
                                border-radius: 5px;
                            </string>
                        </property>
                        <property name="echoMode">
                            <enum>QLineEdit::Password</enum>
                        </property>
                    </widget>
                </item>
                <item row="1" column="2">
                    <widget class="QLabel" name="u_label">
                        <property name="text">
                            <string>Username:</string>
                        </property>
                    </widget>
                </item>
                <item row="8" column="0" colspan="8">
                    <spacer name="bottomSpacer">
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>20</width>
                                <height>40</height>
                            </size>
                        </property>
                    </spacer>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
